src/site/asciidoc/security.adoc

```diff
@@ -46,6 +46,89 @@ If you have encountered an unlisted security vulnerability or other unexpected b
 The threat model that Log4j uses considers configuration files as safe input controlled by the programmer; **potential vulnerabilities that require the ability to modify a configuration are not considered vulnerabilities** as the required access to do so implies the attacker can execute arbitrary code.
 ====
 
+[#model]
+== Threat Model
+
+Log4j is a low level library where configuration inputs and the classpath are expected to be controlled by the programmer.
+Configurations have the ability to execute arbitrary code through custom plugins.
```

Comment:
Arbitrary code can very well be executed in a mere configuration, say, using `ScriptFilter`; using plugins is not a necessity.
```suggestion
Configurations have the ability to execute arbitrary code.
```